rules/Php81/Rector/FuncCall/NullToStrictStringFuncCallArgRector.php

```diff
@@ -488,6 +490,15 @@ private function processNullToStrictStringOnNodePosition(
         return $funcCall;
     }
 
+    private function shouldSkipTrait(Expr $expr, bool $isTrait): bool
+    {
+        if (! $expr instanceof MethodCall) {
```

Comment:
The `MethodCall` or possibly `StaticCall` detection may be in next iteration for improvement where the result is Typed and actually non-mixed type, eg: Nullable type.